### PR TITLE
Src/ Rename

### PR DESCRIPTION
**Pull request by rjwright**
This renames some constructors (and associated local vars/functions) to be in agreement with [recent spec changes](http://w3c.github.io/web-animations/#changes-since-last-publication)

This pull request only contains changes to the src/ directory, plus the bare minimum changes in test/ to keep the tests passing. I have left the tests mostly unchanged for now, for the sake of showing that the old naming still works. A followup PR will change the tests over to the new naming.

@shans @dstockwell PTAL.

### CODE REVIEW

File: src/group-constructors.js [hunk 1]
```diff
@@ -55,32 +55,53 @@
...
   };
 
   scope.groupChildDuration = groupChildDuration;
 
+  // Alias GroupEffect & SequenceEffect to AnimationGroup & AnimationSequence respectively, to
+  // support old constructors (Animation*) for a deprecation period. Should be removed after 23 June
+  // 2015.
+  window.AnimationSequence = function() {
+    if (shared.isDeprecated('window.AnimationSequence', '2015-03-23', 'Use window.SequenceEffect instead.')) {
+      return;
```
Comment (shans) [inline src/group-constructors.js]: probably want to throw some kind of exception here instead of just returning.

File: src/group-constructors.js [hunk 2]
```diff
@@ -55,32 +55,53 @@
...
+    }
+    window.SequenceEffect.apply(this, arguments);
+  };
+  window.AnimationSequence.prototype = Object.create(window.SequenceEffect.prototype);
+  window.AnimationSequence.prototype.constructor = window.AnimationSequence;
+
+  window.AnimationGroup = function() {
+    if (shared.isDeprecated('window.AnimationGroup', '2015-03-23', 'Use window.GroupEffect instead.')) {
+      return;
```
Comment (shans) [inline src/group-constructors.js]: & here

File: src/group-constructors.js (hunk 1 above)
Comment (dstockwell) [inline src/group-constructors.js]: yep, that's what `shared.deprecated` is for

File: src/keyframe-effect-constructor.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+  //
+  // This is only on window and not on scope, because the constructor that was called
+  // webAnimationsNext.Player - now called webAnimationsNext.Animation - is already on the scope.
+  //
+  // FIXME: Add this to scope & expose the other scope.Animation (nee scope.Player). I.e. both this
+  // function and the constructor in web-animations-next-animation should be scope.Animation and
+  // window.Animation until 23 June 2015.
+  window.Animation = function() {
+    if (shared.isDeprecated('window.Animation', '2015-03-23', 'Use window.KeyframeEffect instead.')) {
+      return;
```
Comment (shans) [inline src/keyframe-effect-constructor.js]: throw an exception here.

**Comment by shans** [comment]
LGTM overall. Thanks for taking the time to do this properly!

File: src/group-constructors.js (hunk 1 above)
Comment (rjwright) [inline src/group-constructors.js]: All I have to do is change shared.isDeprecated to shared.deprecated, with exactly the same arguments, right?

**Pull request merged**